cltk/prosody/old_norse/verse.py

```diff
@@ -132,9 +132,9 @@ def to_phonetics(self, transcriber):
         for viisuordh in self.tokenized_text:
             word = normalize(viisuordh)
             if word != "":
```

Comment:
if word !="":
better as:
if word:

Reply:
It is shorter but I am not sure it is easier to understand.
```python
>>> a = ""
>>> bool(a)
False
>>> b = "b"
>>> bool(b)
True
```
As `normalize` returns a string, it is better to check if it is void with `word != ""`, however, this is just an opinion.

Reply:
>>> word=''
>>> if word:
...     print(True)
... else:
...     print(False)
... 
False

Reply:
True:
https://www.python.org/dev/peps/pep-0008/#programming-recommendations
> For sequences, (strings, lists, tuples), use the fact that empty sequences are false.
> Yes:    `if not seq:`
>           `if seq:`
> 
> No:     `if len(seq):`
>           `if not len(seq):`
>